numpy/lib/tests/test_type_check.py

```diff
@@ -278,10 +278,16 @@ def test_generic(self):
 
     def test_integer(self):
         vals = nan_to_num(1)
-        assert_all(vals == 1)
+        assert_all(type(vals) == np.int64)
```

Comment:
This fails on 32 bit linux and windows where the result is 32 bits. Need to take a closer look at this, the test is a bit odd...